charmap.txt

```diff
@@ -311,9 +311,9 @@ WHITE_3 = 41 @ A
 WHITE_4 = 42 @ B
 YELLOW_3 = 43 @ C
 YELLOW_4 = 44 @ D
-LIGHT_BLUE = 45 @ E
+GENDER_COLOR = 45 @ E
```

Comment:
I'm not seeing where this is used for Gender anywhere. I see this is used for `CONTINUE`. How was this name chosen?

Reply:
I got this from MegaMinerd. Apparently it's used as a variable color of blue or pink depending on the player's gender.